### DISCUSSION TITLE
https://www.yunyitang.me/2024-5-update/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#160] [Comments]
# https://www.yunyitang.me/2024-5-update/

很久没有更新了，最近生活多变，其中最好的一件改变就是工作转正。相比平时在小红书刷到的无数招工贴，感觉自己已经很幸运了。 工作两天之后，上手了很不少之前只听同事们谈到的 workflow，感觉离自己想去的职业方向越来越近，还是有点兴奋的。而且手握公司给我这个新驴配的新磨，干啥都顺手了不少。

https://www.yunyitang.me/2024-5-update/

**Comment by Ancvor** [comment]
祝安好

**Reply by yunyit** [reply]
谢谢，同祝好！

**Comment by lkcdm** [comment]
转正了终归是好事，加油，生活在慢慢变好

**Reply by yunyit** [reply]
是呢！一转眼又快转职了（才想起回来休整博客

**Comment by Eternlty** [comment]
👍👍👍
